tools/dftgwbse_CH4/DFTGWBSE_ENERGY.ipynb

```diff
@@ -0,0 +1,759 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DFT + GWBSE Energy Calculation Using CH4"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Requirements\n",
+    "* You will need to install **VOTCA** using the instructions described [here](https://github.com/votca/votca/blob/master/share/doc/INSTALL.md)\n",
+    "* Once the installation is completed you need to activate the VOTCA enviroment by running the `VOTCARC.bash` script that has been installed at the bin subfolder for the path that you have provided for the installation step above"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Interacting with the XTP command line interface\n",
+    "To run a DFT-GWBSE calculation we will use the [xtp_tools](https://votca.github.io/xtp_tools.html) calculator. Run the following command to view the help message of `xtp_tools`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "==================================================\r\n",
+      "========   VOTCA (http://www.votca.org)   ========\r\n",
+      "==================================================\r\n",
+      "\r\n",
+      "please submit bugs to bugs@votca.org\r\n",
+      "\r\n",
+      "xtp_tools, version 1.7-dev gitid: 1c4a17f (dirty) (compiled May 28 2020, 15:47:26)\r\n",
```

Comment:
Is there a way to not store all these outputs?

Reply:
Yes, I am going to redirect the output to a local file, so we just show the command.

Reply:
I mean the output of the cell is automatically used to generate the documentation. Maybe we want to leave some output so the user have an idea what can of output to expect from the program